RecoMTD/TrackExtender/plugins/TrackExtenderWithMTD.cc

```diff
@@ -90,6 +93,62 @@ namespace {
     double timeChi2;
   };
 
+  class TrackSegments {
+  public:
+    TrackSegments() {
+      nSegment_ = 0;
+      segmentPathOvc_.clear();
+      segmentMom2_.clear();
+    }
```

Comment:
```suggestion
    TrackSegments() = default;
```
see the default to 0 suggested below

Reply:
ok